Traceback (most recent call last):
  File "/Users/antonponomarev/PycharmProjects/bitrix_order/venv/lib/python3.10/site-packages/aiogram/dispatcher/middlewares/error.py", line 25, in __call__
    return await handler(event, data)
  File "/Users/antonponomarev/PycharmProjects/bitrix_order/venv/lib/python3.10/site-packages/aiogram/dispatcher/middlewares/user_context.py", line 49, in __call__
    return await handler(event, data)
  File "/Users/antonponomarev/PycharmProjects/bitrix_order/venv/lib/python3.10/site-packages/aiogram/fsm/middleware.py", line 42, in __call__
    return await handler(event, data)
  File "/Users/antonponomarev/PycharmProjects/bitrix_order/venv/lib/python3.10/site-packages/aiogram/dispatcher/event/telegram.py", line 121, in trigger
    return await wrapped_inner(event, kwargs)
  File "/Users/antonponomarev/PycharmProjects/bitrix_order/venv/lib/python3.10/site-packages/aiogram/dispatcher/event/handler.py", line 43, in call
    return await wrapped()
  File "/Users/antonponomarev/PycharmProjects/bitrix_order/venv/lib/python3.10/site-packages/aiogram/dispatcher/dispatcher.py", line 276, in _listen_update
    return await self.propagate_event(update_type=update_type, event=event, **kwargs)
  File "/Users/antonponomarev/PycharmProjects/bitrix_order/venv/lib/python3.10/site-packages/aiogram/dispatcher/router.py", line 142, in propagate_event
    return await observer.wrap_outer_middleware(_wrapped, event=event, data=kwargs)
  File "/Users/antonponomarev/PycharmProjects/bitrix_order/venv/lib/python3.10/site-packages/aiogram/dispatcher/router.py", line 137, in _wrapped
    return await self._propagate_event(
  File "/Users/antonponomarev/PycharmProjects/bitrix_order/venv/lib/python3.10/site-packages/aiogram/dispatcher/router.py", line 170, in _propagate_event
    response = await router.propagate_event(update_type=update_type, event=event, **kwargs)
  File "/Users/antonponomarev/PycharmProjects/bitrix_order/venv/lib/python3.10/site-packages/aiogram/dispatcher/router.py", line 142, in propagate_event
    return await observer.wrap_outer_middleware(_wrapped, event=event, data=kwargs)
  File "/Users/antonponomarev/PycharmProjects/bitrix_order/venv/lib/python3.10/site-packages/aiogram/dispatcher/router.py", line 137, in _wrapped
    return await self._propagate_event(
  File "/Users/antonponomarev/PycharmProjects/bitrix_order/venv/lib/python3.10/site-packages/aiogram/dispatcher/router.py", line 162, in _propagate_event
    response = await observer.trigger(event, **kwargs)
  File "/Users/antonponomarev/PycharmProjects/bitrix_order/venv/lib/python3.10/site-packages/aiogram/dispatcher/event/telegram.py", line 121, in trigger
    return await wrapped_inner(event, kwargs)
  File "/Users/antonponomarev/PycharmProjects/bitrix_order/middleware/outer.py", line 30, in __call__
    return await handler(event, data)
  File "/Users/antonponomarev/PycharmProjects/bitrix_order/venv/lib/python3.10/site-packages/aiogram/dispatcher/event/handler.py", line 43, in call
    return await wrapped()
  File "/Users/antonponomarev/PycharmProjects/bitrix_order/handlers/handler_user.py", line 201, in show_detail_info_order
    await callback.message.answer_media_group(media=media)
  File "/Users/antonponomarev/PycharmProjects/bitrix_order/venv/lib/python3.10/site-packages/aiogram/methods/base.py", line 84, in emit
    return await bot(self)
  File "/Users/antonponomarev/PycharmProjects/bitrix_order/venv/lib/python3.10/site-packages/aiogram/client/bot.py", line 485, in __call__
    return await self.session(self, method, timeout=request_timeout)
  File "/Users/antonponomarev/PycharmProjects/bitrix_order/venv/lib/python3.10/site-packages/aiogram/client/session/base.py", line 254, in __call__
    return cast(TelegramType, await middleware(bot, method))
  File "/Users/antonponomarev/PycharmProjects/bitrix_order/venv/lib/python3.10/site-packages/aiogram/client/session/aiohttp.py", line 189, in make_request
    response = self.check_response(
  File "/Users/antonponomarev/PycharmProjects/bitrix_order/venv/lib/python3.10/site-packages/aiogram/client/session/base.py", line 120, in check_response
    raise TelegramBadRequest(method=method, message=description)
aiogram.exceptions.TelegramBadRequest: Telegram server says - Bad Request: can't parse InputMedia: media not found
